library/data/registry-entry-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/api/package-info.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * <h2>Registry Item Attributes</h2>
+ *
+ * A simple API for adding arbitrary values to any object instances managed by a {@link net.minecraft.util.registry.Registry Registry}.<p>
+ *
+ * {@link org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttribute RegistryEntryAttribute} contains a number of methods for creating
+ * a new attribute.<p>
+ *
+ * To retrieve an attribute value, first get the attribute holder using
+ * {@link org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttributeHolder#get(net.minecraft.util.registry.Registry) RegistryEntryAttributeHolder.get(Registry)},
+ * then use {@link org.quiltmc.qsl.registry.attribute.api.RegistryEntryAttributeHolder#getValue(java.lang.Object, RegistryEntryAttribute)}
+ * to get the value assigned to the specified registry entry.<p>
+ *
+ * <p>
+ *
```

Comment:
Formatting police here. You have two paragraph breaks in a row.
```suggestion
 *
```